.asf.yaml

```diff
@@ -35,6 +35,12 @@ github:
     dev:
       required_status_checks:
         strict: true
+        context:
+          - check-license
+          - CheckDeadLinks
+          - Build
```

Comment:
There is a problem, for some PRs, it only needs to run a part of CI, and here we limit these jobs confirm to be successful, then there may be problems at this time and it cannot be merged

Reply:
Yeah, you're right, maybe we should find some way to mark these checks as success when they were skipped

Reply:
Hi just FYI, GHA getsentry/paths-filter@v2 can be used to filter files changed and set output for later use, example see https://github.com/apache/skywalking-cli/blob/d2c95c3782f137518b09554331794c10e1457d95/.github/workflows/go.yml#L34-L37